packages/patrol/lib/src/custom_finders/patrol_tester.dart

```diff
@@ -79,7 +92,7 @@ class PatrolTesterConfig {
 const defaultScrollDelta = 64.0;
 
 /// Default maximum number of drags during scrolling.
-const defaultScrollMaxIteration = 100;
+const defaultScrollMaxIteration = 15;
```

Comment:
yes, in most cases (as a human) you don't need to do more than a few scrolls, most apps are designed this way. It can be modified, so the 1% of users can change it. I decreased it drastically, bc in case there is no such element that we are scrolling to, the test fails faster.